x/sync/client_test.go

```diff
@@ -144,7 +144,7 @@ func TestGetRangeProof(t *testing.T) {
 	require.NoError(t, err)
 
 	tests := map[string]struct {
-		db                  SyncableDB
+		db                  DB
 		request             *pb.SyncGetRangeProofRequest
```

Comment:
What about these `Sync....Request` types, can these be cleaned up?

Reply:
Yeah, that's a fair point. I left these as is because:
1. When these are used, the import is aliased as `pb`
2. We need a way to disambiguate `SyncGetRangeProofRequest` from `GetRangeProofRequest`

I'm open to other ways of accomplishing (2), though, so if you've got any idea they'd be welcome.

Reply:
I think it's fine as is.. since SyncGetRangeProofRequest & GetRangeProofRequest are both in the same file.